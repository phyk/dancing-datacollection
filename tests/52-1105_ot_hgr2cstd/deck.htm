<!DOCTYPE html PUBLIC "-//WAPFORUM//DTD XHTML Mobile 1.2//EN" "http://www.openmobilealliance.org/tech/DTD/xhtml-mobile12.dtd">
<HTML>
<HEAD>
  <META http-equiv="Content-Type" content="text/html; charset=utf-8">
  <META name="Author" content="Hessischer Tanzsportverband e.V.">
  <META name="GENERATOR" content="TopTurnierDigital">
  <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
  <meta http-equiv="Pragma" content="no-cache" />
  <meta http-equiv="Expires" content="0" />
  <TITLE>11.05.2024 OT, Hgr.II C Standard &#34;7&#34;</TITLE>
  <link rel="stylesheet" type="text/css" href="topturnier.css">
  <meta name="viewport" content="width=device-width, initial-scale=1" />
</HEAD>
<body><div class="mainback">
<div class="eventhead">
<table border=0 width=100%>
<tr><td>11.05.2024 - OT, Hgr.II C Standard &#34;7&#34;</td><td width=30>
<a class="backbtn" href="index.htm" target="_top">&equiv;</a>
</td></tr>
</table>
</div>
<div class="maincontainer">
<div class="comphead">Deckblatt</div>
<hr class="line">
<div class="extract">
   <TABLE class="tab1">
    <TR>
     <TD class="td2" nowrap width=15%>Veranstalter:</TD>
     <TD class="td5" style="word-break:break-all;word-wrap:break-word;">Hessischer Tanzsportverband</TD>
    </TR>
    <TR>
     <TD class="td2" nowrap>Ausrichter:</TD>
     <TD class="td5" style="word-break:break-all;word-wrap:break-word;">Hessischer Tanzsportverband</TD>
    </TR>
    <TR>
     <TD class="td2" nowrap>Turnierleiter:</TD>
     <TD class="td5">
     <span class="col1">Jungbluth, Kai&nbsp;</span> <span>Tanz-Sport-Club Fischbach&nbsp;</span>
     </TD>
    </TR>
    <TR>
     <TD class="td2" nowrap>Beisitzer:</TD>
     <TD class="td5">
     <span class="col1">Knigge, Jens&nbsp;</span> <span>TSC Gro&szlig;-Gerau d. TV 1846&nbsp;</span>
     </TD>
    </TR>
    <TR>
     <TD class="td2" nowrap>Protokoll:</TD>
     <TD class="td5">
     <span class="col1">tanzt, EDV-Team Hessen&nbsp;</span> <span>&nbsp;</span>
     </TD>
    </TR>
    <TR>
     <TD class="td2" nowrap>Wertungsrichter:</TD>
     <TD class="td5"></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>AR:</TD>
     <TD class="td5"> <span class="col1">Appel, Hans-J&uuml;rgen&nbsp;</span> <span>TTC Gelb-Weiss i. Post-SV Hannover&nbsp;</span></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>CH:</TD>
     <TD class="td5"> <span class="col1">Mak, Annabel&nbsp;</span> <span>Gr&uuml;n-Gold-Casino Wuppertal&nbsp;</span></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>DC:</TD>
     <TD class="td5"> <span class="col1">Sch&ouml;ke, Manuel&nbsp;</span> <span>TTC M&uuml;nchen&nbsp;</span></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>DV:</TD>
     <TD class="td5"> <span class="col1">Becker, Marc&nbsp;</span> <span>TTC Fortis Nova Maintal&nbsp;</span></TD>
    </TR>
    <TR>
     <TD class="td2r" nowrap>EY:</TD>
     <TD class="td5"> <span class="col1">Schwarz, Sonja&nbsp;</span> <span>TSZ Blau-Gold Casino, Darmstadt&nbsp;</span></TD>
    </TR>
   </TABLE>
<br>
</div>
</div>
</div>
  <P><FONT size="1" face="Arial">Diese Liste wurde mit <A TARGET="_blank" HREF="http://www.TopTurnier.de">TopTurnier f&uuml;r Windows V9.6b</A> erstellt.<br></FONT>
</body>
</HTML>
