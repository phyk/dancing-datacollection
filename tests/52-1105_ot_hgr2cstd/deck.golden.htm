<!DOCTYPE html>
<html>
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
  <title>
   11.05.2024 OT, Hgr.II C Standard
  </title>
 </head>
 <body>
  <div>
   <div>
    <table>
     <tr>
      <td>
       11.05.2024 - OT, Hgr.II C Standard
      </td>
     </tr>
    </table>
   </div>
   <div>
    <div>
     Deckblatt
    </div>
    <hr/>
    <div>
     <table>
      <tr>
       <td>
        Veranstalter:
       </td>
       <td>
        Hessischer Tanzsportverband
       </td>
      </tr>
      <tr>
       <td>
        Ausrichter:
       </td>
       <td>
        Hessischer Tanzsportverband
       </td>
      </tr>
      <tr>
       <td>
        Turnierleiter:
       </td>
       <td>
        <span>
         Jungbluth, Kai
        </span>
        <span>
         Tanz-Sport-Club Fischbach
        </span>
       </td>
      </tr>
      <tr>
       <td>
        Beisitzer:
       </td>
       <td>
        <span>
         Knigge, Jens
        </span>
        <span>
         TSC Groß-Gerau d. TV 1846
        </span>
       </td>
      </tr>
      <tr>
       <td>
        Protokoll:
       </td>
       <td>
        <span>
         tanzt, EDV-Team Hessen
        </span>
       </td>
      </tr>
      <tr>
       <td>
        Wertungsrichter:
       </td>
      </tr>
      <tr>
       <td>
        AR:
       </td>
       <td>
        <span>
         Appel, Hans-Jürgen
        </span>
        <span>
         TTC Gelb-Weiss i. Post-SV Hannover
        </span>
       </td>
      </tr>
      <tr>
       <td>
        CH:
       </td>
       <td>
        <span>
         Mak, Annabel
        </span>
        <span>
         Grün-Gold-Casino Wuppertal
        </span>
       </td>
      </tr>
      <tr>
       <td>
        DC:
       </td>
       <td>
        <span>
         Schöke, Manuel
        </span>
        <span>
         TTC München
        </span>
       </td>
      </tr>
      <tr>
       <td>
        DV:
       </td>
       <td>
        <span>
         Becker, Marc
        </span>
        <span>
         TTC Fortis Nova Maintal
        </span>
       </td>
      </tr>
      <tr>
       <td>
        EY:
       </td>
       <td>
        <span>
         Schwarz, Sonja
        </span>
        <span>
         TSZ Blau-Gold Casino, Darmstadt
        </span>
       </td>
      </tr>
     </table>
     <br/>
    </div>
   </div>
  </div>
 </body>
</html>
